arm-keyvault/2015-06-01/swagger/keyvault.json

```diff
@@ -0,0 +1,459 @@
+        "tags": [
+          "Vaults"
+        ],
+        "operationId": "Vaults_List",
+        "description": "The List operation gets information about the vaults associated with the subscription.",
+        "parameters": [
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [ "resourceType eq 'Microsoft.KeyVault/vaults'" ],
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "format": "int32",
+            "description": "Maximum number of results to return."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [ "2015-11-01" ],
+            "description": "Azure Resource Manager Api Version."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Get information about all key vaults in the subscription.",
+            "schema": {
+              "$ref": "#/definitions/VaultListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "SKU family name",
+          "enum": [ "A" ],
+          "x-ms-enum": {
+            "name": "SkuFamily",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "SKU name to specify whether the key vault is a standard vault or a premium vault.",
+          "enum": [ "standard", "premium" ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "AccessPolicyEntry": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault."
+        },
+        "objectId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The object ID of a user or service principal in the Azure Active Directory tenant for the vault. "
+        },
+        "applicationId": {
+          "type": "string",
+          "format": "uuid",
+          "description": " Application ID of the client making request on behalf of a principal"
+        },
+        "permissions": {
+          "$ref": "#/definitions/Permissions",
+          "description": "Permissions the identity has for keys and secrets"
```

Comment:
Thanks @tbombach. The description is not ignored in the generated code if used along with $ref. It is used as documentations for constructors and properties. If the description should be removed the autorest should be updated to include the referenced field documentation for properties and constructors. Until autorest is updated, I will keep the documentation as is.